packages/frontend-2/lib/common/helpers/graphql.ts

```diff
@@ -744,14 +748,81 @@ export const modifyObjectField = <
           get: getIfExists,
           evict,
           readField,
-          ref: getObjectReference
+          ref: getObjectReference,
+          fromRef: parseObjectReference
         }
       })
     },
     options
   )
 }
 
+/**
+ * Iterate over object field versions (same field can have different arg versions). If you also
+ * want to make updates, use modifyObjectField instead.
+ */
+export const iterateObjectField = <
```

Comment:
like modifyObjectField but w/o the modification aspect - ultimately the easiest way to just read stuff off cache, better than apollo's own utilities like readFragment and readQuery